cmd/promtool/main.go

```diff
@@ -14,18 +14,123 @@
 package main
 
 import (
+	"flag"
 	"fmt"
 	"io/ioutil"
 	"os"
 	"path/filepath"
+	"strconv"
 	"strings"
+	"time"
 
+	"golang.org/x/net/context"
+
+	"github.com/prometheus/client_golang/api/prometheus"
 	"github.com/prometheus/common/version"
 	"github.com/prometheus/prometheus/config"
 	"github.com/prometheus/prometheus/promql"
 	"github.com/prometheus/prometheus/util/cli"
 )
 
+var (
+	server  = flag.String("server", "http://localhost:9090", "URL of the Prometheus server to query (default http://localhost:9090)")
```

Comment:
ah yes i didn't realize that it prints the default value again, thanks. So should it be a NewFlagSet for each funcs?